cryptol-remote-api/python/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "cryptol"
-version = "3.1.0.99"
+version = "3.1.1.00"
```

Comment:
Use 3.1.1 instead of 3.1.1.00, as the [existing convention](https://pypi.org/project/cryptol/#history) for release versions is to use three positions. (The use of four positions is reserved for development versions, as a fourth position is meant to be an "obvious" indicator that it's a non-release version.)